orchagent/portsorch.cpp

```diff
@@ -3983,3 +3997,22 @@ void PortsOrch::getPortSerdesVal(const std::string& val_str,
         lane_values.push_back(lane_val);
     }
 }
+
+void PortsOrch::flushFDBEntries(sai_object_id_t bridge_port_id)
+{
+    sai_attribute_t attr;
+    vector<sai_attribute_t> attrs;
+    sai_status_t rv;
+
+    SWSS_LOG_INFO("Flushing the port with bridge port id: %" PRIx64, bridge_port_id);
+
+    attr.id = SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID;
+    attr.value.oid = bridge_port_id;
+    attrs.push_back(attr);
+    rv = sai_fdb_api->flush_fdb_entries(gSwitchId, (uint32_t)attrs.size(), attrs.data());
```

Comment:
Missing CRM counter update

Reply:
They are updated on receiving flush notification in function storeFdbEntryState()